core/src/banking_stage/latest_unprocessed_votes.rs

```diff
@@ -185,9 +211,9 @@ impl LatestUnprocessedVotes {
         &'a self,
         votes: impl Iterator<Item = LatestValidatorVotePacket> + 'a,
     ) -> impl Iterator<Item = LatestValidatorVotePacket> + 'a {
-        let staked_nodes = self.cached_staked_nodes.read().unwrap();
+        let epoch_stakes = self.cached_epoch_stakes.read().unwrap();
         votes.filter(move |vote| {
-            let stake = staked_nodes.get(&vote.pubkey()).copied().unwrap_or(0);
+            let stake = epoch_stakes.vote_account_stake(&vote.vote_pubkey());
```

Comment:
This now matches fork choice:
https://github.com/anza-xyz/agave/blob/18059246e5e845855a2d4f8f0441a4fabd101ca6/core/src/consensus/heaviest_subtree_fork_choice.rs#L1049